addons/sevenrings/script_component.hpp

```diff
@@ -0,0 +1,50 @@
+#define COMPONENT seven_rings
+#define COMPONENT_BEAUTIFIED Seven Rings
+#include "\x\tacu\addons\main\script_mod.hpp"
+#include "\x\tacu\addons\main\script_macros.hpp"
+
+#define MACRO_SCOPE \
+    scope = 1; \
+    scopeCurator = 1;
+
+#define MACRO_WEST_DEFAULT_WEAPON \
+    "CUP_hgun_M9", \
+    "Throw", \
+    "Put"
+
+#define MACRO_EAST_DEFAULT_WEAPON \
+    "hgun_Pistol_01_F", \
+    "Throw", \
+    "Put"
+
+#define MACRO_WEST_DEFAULT_MAGAZINE \
+    mag_2("CUP_15Rnd_9x19_M9"), \
+    mag_2("HandGrenade"), \
+    mag_2("MiniGrenade")
+
+#define MACRO_EAST_DEFAULT_MAGAZINE \
+    mag_2("10Rnd_9x21_Mag"), \
+    mag_2("CUP_HandGrenade_RGO"), \
+    mag_2("CUP_HandGrenade_RGD5")
```

Comment:
Ultimately up to you what feels right, but my only "problem" with this method is that everything has the same.
I personally think it's better if the units are varied, eg. medics maybe having less/no lethal grenades but more smokes, team leaders maybe having some colours/flares or something for marking (I know AI doesn't really do that, but at least would allow someone in Zeus to take control and do it sorta manually).

Reply:
From the gameplay perspective i've rarely seen the AI utilise smokes when they're actually useful, most of the time they simply throw them AFTER the engagement ends and block line of sight when they have the advantage. Medics do still carry smokes but overall for AI i find it relatively pointless.